tools/accuracy_checker/accuracy_checker/data_readers/data_reader.py

```diff
@@ -20,6 +20,7 @@
 import re
 import cv2
 import numpy as np
+from scipy.io import wavfile
```

Comment:
scipy is hard dependency for processing on some plathorms (i mean ARM at least). If it is possible, could you please wrap its import to stay it optional if user do not want to work with your reader (see pillow_imread and tf_imread as example)

Reply:
It might be possible to implement this without depending on SciPy by using the builtin `wave` module.

Reply:
Since we already have SciPy in dependencies, I won't reinvent the wheel and continue using `scipy.io.wavfile` instead of `wave`

Reply:
@eizamaliev  it leads to trouble and require invent additional workaround for model validation.
 I want remaind you that this model support requiest have a one smalll yocto arm device as target platform, where no opportunity to build scipy (due to lack of some core libs for it)...

Reply:
Done

Reply:
thanks

Reply:
please fix import order and I think, it is ready to merge.
P.S. please update model support in @vladimir-dudnik  fork, if it is necessary

Reply:
Import order is fixed, changes don't affect anything outside